.gitlab-ci.yml

```diff
@@ -53,27 +53,6 @@ RPM:
           - aws/rhel-9.1-nightly-aarch64
         INTERNAL_NETWORK: "true"
 
-Composer Tests:
-  stage: test
-  extends: .terraform
-  script:
-    - schutzbot/deploy.sh
-    - /usr/libexec/tests/osbuild-composer/image_tests.sh
-  parallel:
-    matrix:
-      - RUNNER:
-          - aws/fedora-35-x86_64
-          - aws/fedora-35-aarch64
-          - aws/fedora-36-x86_64
-          - aws/fedora-36-aarch64
-          - aws/rhel-8.6-ga-x86_64
-          - aws/rhel-8.6-ga-aarch64
-          - aws/rhel-8.7-nightly-x86_64
-          - aws/rhel-8.7-nightly-aarch64
-          - aws/rhel-9.1-nightly-x86_64
-          - aws/rhel-9.1-nightly-aarch64
-        INTERNAL_NETWORK: "true"
-
```

Comment:
I think we could keep these for a while and keep the manifest-db changes "optional", so that we get a sense of the stability of things.

Reply:
alright for me.